policyengine_us/variables/gov/states/or/tax/income/credits/wfhdc/or_wfhdc_eligibility_category.py

```diff
@@ -0,0 +1,60 @@
+from policyengine_us.model_api import *
+
+
+class OregonWFHDCEligibilityCategory(Enum):
+    age_under_3 = "age_under_3"
```

Comment:
change these to not have policy parameters in them

Reply:
YOUNGEST = "age_under_3"
    YOUNG = "age_3_to_6"
    OLD = "age_6_to_13"
    DISABLED_TEENS = "disabled_13_to_18"
    DISABLED_ADULTS = "disabled_18_and_over"
    NONE = "None"
Can I name them like this?

Reply:
this looks good

Reply:
remove parameters from strings too

Reply:
eg
```
YOUNGEST = "Youngest"
```